docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-import/spark-writer.md

```diff
@@ -2,12 +2,12 @@
 
 ## Overview
 
-Spark Writer is Nebula Graph's Spark-based distributed data import tool that converts data from multiple data repositories into vertices and edges of graphs and batch imports data into the graph database. Currently supported data repositories are:
+Spark Writer is a Spark-based distributed data importer for [Nebula Graph](https://github.com/vesoft-inc/nebula). It converts data from multiple data sources into vertices and edges for graphs and batch imports data into the graph database. Currently supported data sources are:
```

Comment:
Currently, the supported data sources are:

Or change it to "It supports source data of these formats:"